arangod/GeneralServer/AsyncJobManager.cpp

```diff
@@ -213,7 +213,8 @@ Result AsyncJobManager::clearAllJobs() {
 
   for (auto const& it : _jobs) {
     bool ok = true;
-    RestHandler* handler = it.second.second._handler;
+    std::shared_ptr<RestHandler> handler = it.second.second._handler;
```

Comment:
Use a reference here to avoid incrementing/decrementing the ref count (which are expensive atomic operations).